sympy/solvers/solveset.py

```diff
@@ -988,36 +988,6 @@ def _solveset(f, symbol, domain, flags={'tsolve_saw': []}, _check=False):
     return result
 
 
-def _term_factors(f):
```

Comment:
Removed this as it wasn't needed due to the changes made in `_is_exponential()` below